source/fundamentals/stable-api.txt

```diff
@@ -0,0 +1,150 @@
+.. _golang-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The {+stable-api+} feature requires MongoDB Server 5.0 or later.
+
+   You should only use the {+stable-api+} feature if all the MongoDB
+   servers you are connecting to support this feature.
+
+Overview
+--------
+
+In this guide, you can learn how to specify the **{+stable-api+}** when
```

Comment:
**Suggestion**: to make it clearer in this intro sentence what it means to specify the Stable API, I would change this slightly to include "compatibility"
```suggestion
In this guide, you can learn how to specify **{+stable-api+}** compatibility when
```